go.mod

```diff
@@ -195,3 +195,5 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/leonklingele/grouper v1.1.1 => github.com/alexandear/grouper v1.1.2
```

Comment:
Must be removed after merging https://github.com/leonklingele/grouper/pull/4 and creating a new release

Reply:
sorry but the dependencies are handled by the `go.mod` of golangci-lint, an update to the dependencies of a linter are not required.

Reply:
Also, I recommend never creating a "custom" tag for a dependency.
You don't need it: you can use hash or branch name.